### PR TITLE
Add summary_sixty_char metadata for all specs

### PR DESCRIPTION
**Pull request by starksm64**
Signed-off-by: Scott M Stark <starksm64@gmail.com>

### CODE REVIEW

File: annotations/_index.md [hunk 1]
```diff
@@ -2,5 +2,7 @@
 title: "Jakarta Annotations"
 summary: "Jakarta Annotations defines a collection of annotations representing common semantic concepts that
 enable a declarative style of programming that applies across a variety of Java technologies."
+<!--.................0123456789.123456789.123456789.123456789.123456789.123456789-->
+summary_sixty_char: "Declarative style snnotations for common semantic concepts"
```
Comment (kwsutter) [inline annotations/_index.md:6]: annotations

File: xml-web-services/_index.md [hunk 2]
```diff
@@ -2,5 +2,7 @@
 title: "Jakarta XML Web Services Specification"
 summary: "Jakarta XML Web Services defines a means for implementing XML-Based Web Services
 based on Jakarta SOAP with Attachments and Jakarta Web Services Metadata."
+<!--.................0123456789.123456789.123456789.123456789.123456789.123456789-->
+summary_sixty_char: "defines a means for XML-Based Web Services based on SOAP"
```
Comment (kwsutter) [inline xml-web-services/_index.md:6]: Defines (capital)

**Review by kwsutter — APPROVED** [review]
Two nits.  Thank you, @starksm64!

**Review by arjantijms — APPROVED** [review]

**Comment by arjantijms** [comment]
Good summary for the specs I'm involved with, thanks!

File: restful-ws/3.0/_index.md [hunk 3]
```diff
@@ -33,19 +33,20 @@ Please reference that ballot for the official results.
```
Comment (kwsutter) [inline restful-ws/3.0/_index.md:33]: Why was this update added to this PR?  I think we have a separate PR #300 for jax-rs ballot results.

**Review by kwsutter — APPROVED** [review]
I think there was an errant commit to this branch...  I'll still Approve, but we need to ensure that we're not accidentally overwriting information from other commits on other PRs....

File: restful-ws/3.0/_index.md (hunk 3 above)
Comment (starksm64) [reply restful-ws/3.0/_index.md]: Should be fixed, I forgot which repo I was using for this change

**Comment by starksm64** [comment]
It looks like the last commit fixed the issue with the content build checks failing.

**Review by kwsutter — APPROVED** [review]
Thanks, @starksm64!

**Pull request merged**